axolotl-web/src/assets/images/loading.svg

```diff
@@ -0,0 +1,11 @@
+<!--  SVG by Sam Herbert (@sherb), for everyone. More @ http://goo.gl/7AJzbL  -->
```

Comment:
This is the same SVG that in `index.html` but I needed it as an image file this time.

Reply:
that's fine